praw/models/subreddits.py

```diff
@@ -80,6 +80,17 @@ def search_by_name(self, query, include_nsfw=True, exact=False):
                                          'exact': exact, 'query': query})
         return [self._reddit.subreddit(x) for x in result['names']]
 
+    def search_by_topic(self, query):
```

Comment:
Yes, super easy! Thanks for writing a really easy to read and use library! :100: